src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -32,11 +32,11 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
                 const config = state.config.getStoreConfig();
                 const cart = state.cart.getCart();
 
-                if (!config) {
+                if (!config || !config.currency) {
                     throw new MissingDataError(MissingDataErrorType.MissingCheckoutConfig);
                 }
 
-                if (!cart) {
+                if (!cart || !cart.id) {
```

Comment:
Same here, `cart.id` is not defined as an optional so you can also omit the null check here.

Reply:
Thank You!